maplibre-native-ios/Amazon Location Service Demo/MapView.swift

```diff
@@ -5,25 +5,27 @@ import AWSCore
 import Mapbox
 import SwiftUI
 
+func configureMapbox() {
+    // Workaround for https://github.com/aws-samples/amazon-location-samples/issues/31
```

Comment:
Workaround or solution? I'd argue the latter, esp. if you don't feel that this is a hacky solution.